commands/meme.js

```diff
@@ -0,0 +1,43 @@
+const Augur = require("augurbot"),
+    u = require("../utils/utils");
+
+const Module = new Augur.Module()
+    .addCommand({
+        name: "meme",
+        description: "Creates a meme, put an image URL for you background and then put the text you want along the bottom. Or put the image source afterwards. Who am I to judge?",
+        process: (msg) => {
+            //general globals from bot this was imported from
+            let regex = /[!"#$%&'()*+,./:;<=>?@[\]^_`{|}~]/g;
+            const args = message.content.slice(prefix.length).trim().split(/ +/);
```

Comment:
A couple things with this line. First, `prefix` isn't defined directly, it would be `Module.config.prefix` *if* we're to do it this way.

Second, either the `!` prefix *or* the `@Icarus` mention can work as a prefix, which means doing things this way can yield unexpected results. I suggest using the `suffix` parameter introduced on line 8. This will have everything trailing the command.
```suggestion
            const args = suffix.trim().split(/ +/);
```